--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,7 +68,7 @@
         src/provider/CloudSSOCredentialsProvider.cpp
         src/provider/OAuthCredentialsProvider.cpp)

-add_library(${PROJECT_NAME} SHARED ${SOURCE_FILES})
+add_library(${PROJECT_NAME} STATIC ${SOURCE_FILES})

 # Ensure main library uses dynamic runtime library on Windows
 if(MSVC)
--- a/external/darabonba_core/CMakeLists.txt
+++ b/external/darabonba_core/CMakeLists.txt
@@ -26,6 +26,9 @@
       -DCMAKE_C_FLAGS_DEBUG:STRING="/MDd")
   endif()

+  list(APPEND _darabonba_core_CMAKE_ARGS
+    -DCMAKE_CXX_FLAGS:STRING="${CMAKE_CXX_FLAGS} -DEVP_MD_CTX_get0_md=EVP_MD_CTX_md -fvisibility=hidden -fvisibility-inlines-hidden -Wno-error -Wno-defaulted-function-deleted -Wno-unused-but-set-parameter")
+
   FetchContent_Declare(
     _darabonba_core
     GIT_REPOSITORY https://github.com/aliyun/tea-cpp.git