--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,1 +71,1 @@
-add_library(${PROJECT_NAME} SHARED ${SOURCE_FILES})
+add_library(${PROJECT_NAME} STATIC ${SOURCE_FILES})
--- a/external/darabonba_core/CMakeLists.txt
+++ b/external/darabonba_core/CMakeLists.txt
@@ -10,1 +10,2 @@
   set(_darabonba_core_CMAKE_ARGS
     -DENABLE_UNIT_TESTS:BOOL="0")
+    -DCMAKE_CXX_FLAGS:STRING="${CMAKE_CXX_FLAGS} -DEVP_MD_CTX_get0_md=EVP_MD_CTX_md")