--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -3,1 +3,1 @@
-add_library(${PROJECT_NAME} SHARED ${CPP_FILES})
+add_library(${PROJECT_NAME} STATIC ${CPP_FILES})
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,1 +28,1 @@
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -DEVP_MD_CTX_get0_md=EVP_MD_CTX_md")
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,1 +32,1 @@
-option(ENABLE_UNIT_TESTS "Enable unit tests" ON)
+option(ENABLE_UNIT_TESTS "Enable unit tests" OFF)